stencil/src/components/pulumi-api-doc-filterable-nav/pulumi-api-doc-filterable-nav.scss

```diff
@@ -0,0 +1,43 @@
+@import "../../../../src/scss/colors";
+
+.api-doc-nav {
```

Comment:
This context is super helpful, thank you!  Makes a lot of sense, and I certainly didn't intend to intentionally diverge from the established patterns.  I would love to merge this as-is, simply because it's blocking me from tackling related items, but agree that it should be addressed in a follow-up as soon as we have hit the demo capabilities milestone.  I'll track that here, and assign to myself:  https://github.com/pulumi/theme/issues/13.  I'm going to resolve the other related comments, since the above follow-up will take care of them as well.

>  I wanted to allow for the possibility of pulling our components up into a library at some point, so I tried to keep the visual and behavioral styling pretty separate, thinking a user of this hypothetical future component library might want the functionality of a language chooser, say, but not be forced to use the visual styles of pulumi.com for it.

I do want to note that this is actually a very different way of thinking about a component library than I had been thinking about it.  In my mind, one benefit of a component library is to ensure a shared visual vocabulary that does impose styles (or options that force adherence to one of several different style sets) on the consumer.  Not a discussion we need to pull into this PR, but I did want to note that it's an approach that's a little different than I'd imagined - maybe a great item for us to discuss as a larger team in that UI meeting you set up for post-CES.